stories/Select.tsx

```diff
@@ -68,4 +73,68 @@ storiesOf('Organisms|Select', module)
                 />
             </>
         );
+    })
+    // eslint-disable-next-line
+    .add('AutosuggestMulti', ({ parameters }: any) => {
+        const store = parameters.getStore();
+        searchFor.name = `Search for "${store.get('inputValue')}"`;
+        searchFor.value = store.get('inputValue');
+        const getSuggestions = () => {
+            if (!store.get('inputValue').length) return [];
+            const suggestions = SUGGESTIONS.filter(
+                (item) => !store.get('selectedSuggestions').includes(item)
```

Comment:
checking equality for `item` doesn't work, because it is an object. You need to check for `item.name`

Reply:
it works :) It will return true if we have a link on the same object.
The same checks we have for 'Autosuggest' and `ComboboxMulti`

Reply:
<img width="670" alt="Screenshot 2020-03-27 at 09 58 17" src="https://user-images.githubusercontent.com/23654472/77739367-aef0e800-7011-11ea-8ea4-1ba9868d65bd.png">

From the code, I thought that selected items should not be available to be reselected.